src/OperatorMsg.chpl

```diff
@@ -1838,6 +1839,21 @@ module OperatorMsg
                     }
                 }
             }
+            when (DType.Bool, DType.Bool) {
+                var l = toSymEntry(left, bool, nd);
+                var val = value.getBoolValue();
+                select op {
+                    when "+=" {
+                      ref la = l.a;
+                      [li in la] la = if (li == true || val == true) then true else false;
+                    }
```

Comment:
I think this could also be `l.a |= val`?

Reply:
My bad, left this one out, great catch